src/Faker/Provider/it_IT/Sport.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Faker\Provider\it_IT;
+
+class Sport extends \Faker\Provider\Sport
+{
+    protected static $sports = array(
+        'Golf',
```

Comment:
As @OskarStark suggested, can you sort alphabetically by value here?

Reply:
Couldn't this be fixed through the whole codebase by a CS fixer rule?

Reply:
I think http://plugins.jetbrains.com/plugin/5919-lines-sorter will work best here.